doltpy/cli/dolt.py

```diff
@@ -166,14 +167,11 @@ class Dolt:
     launches an interactive shell.
     """
 
-    def __init__(self, repo_dir: str, server_config: ServerConfig = ServerConfig()):
+    def __init__(self, repo_dir: str):
```

Comment:
option for quietly failing calls maybe? maybe just a decorator that injects kwarg and wraps with a try/except

Reply:
or have you tried stdout/stderr handling?

Reply:
What do you mean quietly failing calls, not sending errors to stdout?

Reply:
i) not having a way to quiet a `Dolt.init()` was a pain in demos
ii) i was thinking the subprocess **kwargs give users control over what happens with stdout/stderr, definitely not a priority but allowing flow-through kwargs might be easy

Reply:
Agreed with this, breaking out the SQL stuff should ensure no noisy SQL server logs, we can address the configurations exposed to he user in due course. Agreed we want to tidy that up with sensible defaults.